docker-compose.yml

```diff
@@ -38,6 +38,7 @@ services:
     ports:
       - "7770-7771:7770-7771"
     image: c4dt/byzcoin:latest
+    restart: always
```

Comment:
Shouldn't this be `on-failure` or `unless-stopped`? I'm not sure if you still can do `docker-compose down` when it is on `always`...

Reply:
According to [this](https://docs.docker.com/config/containers/start-containers-automatically/), it would stop it until the Docker daemon restarts.

Reply:
OK, then it should definitely be `unless-stopped`, or `on-failure`.